IHP/IDE/Defaults/TableColumnDefaults.hs

```diff
@@ -0,0 +1,232 @@
+{-|
+Module: Test.IDE.Defaults.TableColumnDefaults
+Copyright: (c) digitally induced GmbH, 2020
+-}
+module IHP.IDE.Defaults.TableColumnDefaults where
+
+import Test.Hspec
+import IHP.Prelude
+import qualified IHP.IDE.CodeGen.ControllerGenerator as ControllerGenerator
+import IHP.ViewPrelude (cs, plain)
+import qualified Text.Megaparsec as Megaparsec
+import IHP.IDE.CodeGen.Types
+import IHP.IDE.SchemaDesigner.Types
+import IHP.NameSupport
+import IHP.IDE.SchemaDesigner.Types
+
+
+{- | Takes a 'Text' value for the name and creates a default column where all values are empty lists, 
+'Nothing' or 'False'; and for 'ColumnType', the default is 'PUUID'.
+
+Add a new field to the 'Column' type in the 
+"IHP.IDE.SchemaDesigner.Types" file and then 
+set its default value here
+
+Defined as:
+
+@
+defColumn :: Column
+defColumn = Column { 
+                 name = ""
+               , columnType = PUUID 
+               , defaultValue = Nothing
+               , notNull = False 
+               , isUnique = False
+               , generator = Nothing
+               }
+@
+
+If you want a different 'PostgresType' you will need to 
+specify with like so by either using a function:
+
+@setColumnType pgt = defColumn {columnType = pgt}@
```

Comment:
Some of the comments here are outdated, e.g. `setColumnType` doesn't exist anymore. Also there's sometimes still some markdown syntax \`something\` instead of 'something' .